config/app.default.php

```diff
@@ -257,7 +257,28 @@
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
 
-            'url' => env('DATABASE_URL', null),
+            'url' => env('DATABASE_URL', 'chainquery.lbry.io:3600'),
```

Comment:
@marcdeb1 Please change the DSN string to `mysql://chainquery.lbry.io:3600` as this is causing the Travis build to fail. Thanks.